docs/guide/useEnvVar.md

```diff
@@ -0,0 +1,51 @@
+# Environment Variables
+
+If you wanna use Environment Variables in your callback's argument, like an API key or anything else, you need to follow the next steps.
+
+## Create a `hygie-env` repo
+
+First of all, you need to create a **PRIVATE** `hygie-env` git repository at the **same level than your repo(s) using `Hygie`**.
```

Comment:
'same lavel' ? we can have multi level in an orga ? You should say something like 'in the same organization of your repo'.